Content.Client/Playtime/ClientsidePlaytimeTrackingManager.cs

```diff
@@ -66,11 +76,19 @@ private void OnConnected(object? sender, NetChannelArgs args)
         var recordedDateString = _configurationManager.GetCVar(CCVars.PlaytimeLastConnectDate);
         var formattedDate = datatimey.Date.ToString(InternalDateFormat);
 
+        // Moffstation - Start - Hourly Playtime Notice
         if (formattedDate == recordedDateString)
+        {
+            // Still reschedule the hourly notice to avoid duplicate timers across reconnects.
+            ScheduleNextHourlyNotice();
             return;
+        }
+        // Moffstation - End - Hourly Playtime Notice
 
         _configurationManager.SetCVar(CCVars.PlaytimeMinutesToday, 0);
         _configurationManager.SetCVar(CCVars.PlaytimeLastConnectDate, formattedDate);
+
+        ScheduleNextHourlyNotice(); // Moffstation - Hourly Playtime Notice
```

Comment:
Scheduling the notice doesn't use the `CCVars`, so why not put this before the `if` and avoid the duplication?